### DISCUSSION TITLE
RBAC enablement at namespace level throws constant error

### DISCUSSION THREAD

**Discussion by arpitjain1** [chaos-mesh/chaos-mesh#3282] [Q&A] [answered]
**Summary:**

We are trying to enable RBAC at namespace level and it throws constant error regarding cluster level access.

**Details:**
Namespace1: chaos-testing (where chaos-mesh daemons are deployed)
Namespace2: service1 (where namespace level access needs to be defined)
Note: chaos mesh was installed following https://chaos-mesh.org/docs/production-installation-using-helm/.

**Reference used:** 
https://chaos-mesh.org/docs/ma
<img width="754" alt="error" src="https://user-images.githubusercontent.com/15323151/169544827-6e83e552-8c9a-408a-9923-0b671fa757fc.png">
nage-user-permissions/
_Here, rbac.yaml has been applied on service1 namespace, as it wasn't clear which namespace this should be applied to._

**Problem:**
1. Constant error complaining about cluster level access (screenshot attached)
2. Experiments was not getting saved when tried with manager token, even not in execution history. It was getting executed though.

Need help in resolving above problems and make rbac work at namespace level.

**Comment by WangXiangUSTC** [comment, marked as answer]
You can choose the namespace here, is it "ALL" in your dashboard?
![image](https://user-images.githubusercontent.com/5793595/169826779-17d142ad-d2f5-4ff3-acfc-fcb52f8f4f30.png)
